server/queue/queue.php

```diff
@@ -15,11 +15,12 @@ function loadPublicQueue($data) {
 	GLOBAL $DB_CONN;
 	$query = "SELECT Appointment.appointmentId, TIME_FORMAT(scheduledTime, '%l:%i %p') AS scheduledTime, firstName, lastName, 
 				timeIn, timeReturnedPapers, timeAppointmentStarted, timeAppointmentEnded, completed,
-				(DATE_ADD(AppointmentTime.scheduledTime, INTERVAL 15 MINUTE) < NOW() AND timeIn IS NULL) AS noshow 
+				(DATE_ADD(AppointmentTime.scheduledTime, INTERVAL 15 MINUTE) < NOW() AND timeIn IS NULL) AS noShow,
+				(AppointmentTime.scheduledTime < Appointment.createdAt) AS walkIn
```

Comment:
I would love a comment on this design. I'm not certain running this comparison in the SQL statement is the best idea? We could easily do this on the front-end or in PHP instead if we returned the unformatted scheduledTime and createdAt. Thoughts from a best practices perspective?